packages/spanner/src/v1/index.js

```diff
@@ -26,10 +26,9 @@ function v1(options) {
   var gaxGrpc = gax.grpc(options);
   return spannerClient(gaxGrpc);
 }
+
+v1.GAPIC_VERSION = '0.0.5';
 v1.SERVICE_ADDRESS = spannerClient.SERVICE_ADDRESS;
 v1.ALL_SCOPES = spannerClient.ALL_SCOPES;
-v1.admin = {
```

Comment:
This going away breaks our handwritten Spanner constructor:

```js
function Spanner(options) {
  // ...

  this.api = {
    Database: v1.admin.database(options).databaseAdminClient(options),
    Instance: v1.admin.instance(options).instanceAdminClient(options),
    Spanner: v1(options).spannerClient(options)
  };
```

Is there a new way to get to those clients?

Reply:
No, this looks like something that was special for Spanner and autogen does not know that. I will set it back for now. :-)